prog/test/ha_postgres_resource.rb

```diff
@@ -0,0 +1,150 @@
+# frozen_string_literal: true
+
+require_relative "../../lib/util"
+
+class Prog::Test::HaPostgresResource < Prog::Test::Base
+  semaphore :destroy
+
+  def self.assemble
+    postgres_test_project = Project.create(name: "Postgres-HA-Test-Project")
+    postgres_service_project = Project[Config.postgres_service_project_id] ||
+      Project.create(name: "Postgres-Service-Project") { _1.id = Config.postgres_service_project_id }
+    minio_service_project = Project[Config.minio_service_project_id] ||
+      Project.create(name: "Minio-Service-Project") { _1.id = Config.minio_service_project_id }
+
+    frame = {
+      "postgres_service_project_id" => postgres_service_project.id,
+      "postgres_test_project_id" => postgres_test_project.id,
+      "minio_service_project_id" => minio_service_project.id,
```

Comment:
We don't need to store minio_service_project_id and postgres_service_project_id, right? They are already in `Config`.